### DISCUSSION TITLE
masked token prediction

### DISCUSSION THREAD

**Discussion by BHX2** [explosion/spaCy#7443] [Help: Coding & Implementations] [answered]
Is it possible to use the spacy-transformer model for prediction of `<mask>` tokens? I had originally planned to use the pytorch shim for roberta-base that I thought would be within en_core_web_trf but it seems that the thinc model does not include the pytorch model within it. My understanding is that only the converted model is kept. If so, would a forward pass with injected `<mask>` tokens predict probabilities similar to the original pytorch transformer?

**Comment by polm** [comment, marked as answer]
Sorry for the delayed reply, but spaCy has no features for this. This is considered an NLG task and NLG is considered out of scope for spaCy.
